lib/maze/configuration.rb

```diff
@@ -96,6 +96,20 @@ def add_validator(endpoint, &validator)
       @custom_validators[endpoint] = validator
     end
 
+    # Whether default validation should be skipped for a given endpoint
+    attr_reader :skipped_validators
+
+    # Sets whether to skip default validation for a given endpoint
+    #
+    # @param endpoint [String] The endpoint to skip default validation for
+    def skip_default_validation(endpoint)
```

Comment:
Passing the endpoint as a string is kind of brittle - is it "error" or "errors", for example.  That's not the end of the world as it will only mean false failures not false passes, I just wonder if we should create ticket to introduce an enum type for the endpoints and apply it across the codebase (not part of this PR).